src/api/app/helpers/application_helper.rb

```diff
@@ -1,3 +1,8 @@
 # Methods added to this helper will be available to all templates in the application.
 module ApplicationHelper
+  def render_diff(content)
+    # rubocop:disable Rails/OutputSafety
+    CodeRay.scan(content, :diff).div(line_numbers: :table, css: :class).html_safe
```

Comment:
Hmmmmmm. Diffs are user input right?

Reply:
Yes

Reply:
@hennevogel your comments is regarding user input validation x marking html as safe?

Reply:
yeah, if I can create whatever diff I want  and you mark it as html safe, can I inject XSS?

Reply:
but did you try to inject it? I tried it and in a `pre` tag didn't work. I tried too to close the `pre` tag as `</pre><script>alert('foo')</script>` and didn't work too.  Any idea @hennevogel ?

Reply:
nope, if you tried, all is fine.